src/en/02_quick_start.md

```diff
@@ -15,20 +17,61 @@ chmod +x ./pgmanage-$version.AppImage
 
 ### Windows
 
-Run the PgManage setup executable and follow the installation instructions.
-
-**FIXME:** add PostgreSQL command line tools installation instructions
+- Run the PgManage setup executable and follow the installation instructions.
+- To install PostgreSQL client utilities, follow the steps in the [Installing Client Utilities on Windows](#installing-client-utilities-on-windows) section.
 
 ### Mac
 
 Download the `DMG` file and double click on it and drag the PgManage icon to the `Applications Folder` icon.
 
 **FIXME:** add PostgreSQL command line tools installation instructions
 
+---
+
+### PostgreSQL Client
+
+When PgManage starts it will try to automatically find PostgreSQL's client binaries, which contain executable files for the `pg_dump`, `pg_restore`, `pg_dumpall`, and `psql` commands. For cases in which this autodiscovery does not work or is not desired, a path to the binaries may be specified on the application’s `Utilities Menu → Settings`.
+
+![Image of the settings dialog](./images/pgmanage-settings.png)
+
+To check that the binaries were found, you may click the `verify` button which will display the installed PostgreSQL’s version.
+
+![Image of the confirmation message](./images/pgmanage-validate.png)
+
+> **Note:** The autodiscovery of client binaries is not available on Windows. The only way to use the backup and restore features is to manually install the PostgreSQL client utilities.
+
+#### Installing Client Utilities on Windows
+
+- You may download a Windows’ PostgreSQL installer from  [enterprisedb.com](https://www.enterprisedb.com/downloads/postgres-postgresql-downloads).
+- Take note of the installation path where the components will be installed.
+
+![Image of the PostgreSQL installer asking for an installation path](./images/edb_windows_psql_installer1.png)
+
+- The installer will ask what components to install. Only install the `command line tools`.
+
+![Image of the PostgreSQL installer asking what components to install](./images/edb_windows_psql_installer2.png)
+
+- Finally, add the path to the binaries in the PgManage’s `Utilities Menu → Settings`. Notice that the binary files are inside the `bin` folder.
+
+![Image of the settings dialog](./images/pgmanage_windows_binaries_validation.png)
+
+#### Installing Client Utilities on Linux:
+
+- Follow the instructions to install PostgreSQL for your particular linux distribution on [postgresql.org](https://www.postgresql.org/download/linux/).
+- Add the path of the binaries in `Utilities Menu → Settings`. For example:
```

Comment:
Setting up paths to postgresql client binaries is not necessary for Linux and Mac. On these platform postgres client utilities usually present in system $PATH environment variable and can be "auto-discovered".
The main use case for setting up binary path manually on linux and mac is:
to point the app to a specific set of postgresql client binaries (in case if multiple versions are installed simultaneously)
to provide the binary path if auto-discovery failed, for example if Postgres client utilities were installed in some non-standard location, which is not included in system $PATH environment variable